include/core/parameters.h

```diff
@@ -401,6 +446,15 @@ namespace Parameters
     unsigned int              number_of_solids;
     static const unsigned int max_solids = 1;
 
+    // Fluid-fluid or fluid-solid interactions
+    std::vector<MaterialInteractions> material_interactions;
+    unsigned int                      number_of_material_interactions;
+    static const unsigned int         max_material_interactions = 5;
```

Comment:
I am unsure why you put five here if you can only have two fluids. I would maybe define this parameter base on the number of fluids.